.github/PULL_REQUEST_TEMPLATE/release_template.md

```diff
@@ -0,0 +1,25 @@
+## Pelorus <VERSION> release
+
+<!-- Write this section if Pull Request has linked issues -->
+## Linked Issues
+
+<!-- Use this if merging should auto-close an issue, one issue per line -->
+resolves #<!-- issue number -->
+
+<!-- Use this if merging should NOT auto-close an issue, one issue per line -->
+related to #<!-- issue number -->
+
+## Requirements for release
+
+- [ ] Update and review GitHub release body in https://github.com/dora-metrics/pelorus/releases
+- [ ] Test Pelorus Operator
+- [ ] Pelorus Operator in OpenShift Marketplace
+- [ ] GitHub release
```

Comment:
Same here. GitHub release happens after this PR is merged.

Reply:
Maybe this should be an issue to track actions, not a PR template?

Reply:
yeah, going to to that